qiskit_ibm_runtime/estimator.py

```diff
@@ -28,7 +27,7 @@
 # TODO import BaseEstimator and EstimatorResult from terra once released
 from .qiskit.primitives import BaseEstimator, EstimatorResult
 from .qiskit_runtime_service import QiskitRuntimeService
-from .program.result_decoder import ResultDecoder
+from .estimator_result_decoder import EstimatorResultDecoder
```

Comment:
We can probably put the new classes into a `decoder` or even the existing `utils` folder to make it more organized. Users won't import these directly anyway.